ansys/dpf/post/dpf_path.py

```diff
@@ -0,0 +1,75 @@
+"""Module containing the DpfPath class that will
+define a path of coordinates to set the result on.
+"""
+import numpy as np
+
+from ansys.dpf.core.common import locations, natures
+from ansys.dpf.core import Field
+
+def create_path_on_coordinates(coordinates):
+    """
+    Create a dpf path object that can be used to request
+    results on a specific path of coordinates.
+
+    Parameters
+    ----------
+        coordinates : list[list[int]], Field, arrays
```

Comment:
one tab too many here 
```suggestion
    coordinates : list[list[int]], Field, arrays
```